gitpod-io/gitpod
# code. Build stable image for 1.67.2

Prepares for the release of https://github.com/microsoft/vscode/releases/tag/1.67.2.

## How to test

1. Select VS Code Web Insiders as the IDE
1. Open workspace
2. Check commit is `5ac63c9924b00a1a7adfe0f879f81a98e8860216`

## Release Notes

```release-note
NONE
```
